mantidproject/mantid
# Time indexes for detector table

This adds a column showing the time indexes to the detector table. The logic is the same as for detector IDs - if there are more than 10 entries then the indexes are just displayed as `1, 2...(100 more)...102, 103`.

**To test:**

Please use the changes in #19728 to test this branch. `CreateSampleWorkspace` now has the option to create a workspace with mulitple time indexes, and `GroupDetectors` can be used to test workspaces with more than one time index.

1. A workspace with no time indexing should not show the column at all
1. A workspace with less a single time index should show the time index
1. A workspace with less than 10 time indexes should show in a comma separated list
1. A workspace with more than 10 time indexes should show in a list such as `1, 2...(100 more)...102, 103`

Fixes #19380.

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
